lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
The error you posted looks weird, `'{ include: { all: boolean; include: { all: boolean; }[]; }[]; }` doesn't look like a valid input to test?